app/lib/uptime_web/live/dashboard_live.ex

```diff
@@ -0,0 +1,72 @@
+defmodule UptimeWeb.DashboardLive do
+  @moduledoc false
+  use UptimeWeb, :live_view
+
+  alias Uptime.Services
+
+  # Placeholder for dashboard setting
+  @show_checks_since DateTime.add(DateTime.utc_now(), -24, :hour)
+  @show_checks_until DateTime.utc_now()
+
+  def render(assigns) do
+    ~H"""
+    <.app>
```

Comment:
is there a reason we wouldn't just leave the app pieces in app.html as the layout?

Reply:
I think I did it this way to customize the header per page if we want, but I haven't really sketched any designs yet that require that. I guess this felt more flexible here.